File: backend-node/src/common/loan-limits.ts
// PRDF product loan limits. Keep in sync with client-ui/src/lib/loanLimits.ts
// and the loan_applications range constraints in infra/supabase/schema.sql.
export const LOAN_AMOUNT_MIN = 250_000;
export const LOAN_AMOUNT_MAX = 5_000_000;
export const LOAN_TERM_MIN = 1;
export const LOAN_TERM_MAX = 60;

File: backend/src/PRDF.Lms.Domain/LoanLimits.cs
namespace PRDF.Lms.Domain;

/// <summary>
/// PRDF product loan limits. Keep in sync with client-ui/src/lib/loanLimits.ts
/// and the loan_applications range constraints in infra/supabase/schema.sql.
/// </summary>
public static class LoanLimits
{
    public const decimal AmountMin = 250_000m;
    public const decimal AmountMax = 5_000_000m;
    public const int TermMonthsMin = 1;
    public const int TermMonthsMax = 60;
}
